client/js/App.js

```diff
@@ -2,12 +2,15 @@ import React, {Component} from 'react';
 import {ApolloProvider} from 'react-apollo';
 import client from './config/api';
 import RootStackNavigator from '../js/navigation/RootStackNavigator';
+import {SaveProvider} from './context/SaveContext';
 
 export default class App extends Component {
   render() {
     return (
       <ApolloProvider client={client}>
-        <RootStackNavigator />
+        <SaveProvider>
+          <RootStackNavigator />
+        </SaveProvider>
```

Comment:
What is `SaveProvider`?

Reply:
It's like FaveProvider in r10, but designers named it "Save", so I named it as SaveProvider

it stores the id for saved events and self-guided tours

Reply:
Fixed